package.json

```diff
@@ -5,26 +5,35 @@
   "main": "dist/index.js",
   "scripts": {
     "build": "ncc build src --license licenses.txt",
-    "lint": "eslint ."
```

Comment:
please restore the lint script, it should be ran as part of the new ci.yml workflow